cloudevents/sdk/http/__init__.py

```diff
@@ -82,5 +83,32 @@ def from_http(
     return CloudEvent(attrs, event.data)
 
 
-def from_json():
-    raise NotImplementedError
+def to_json(
+    event: EventClass, data_marshaller: types.MarshallerType = None
+) -> typing.Union[str, bytes]:
+    """
+    Cast an EventClass into a json object
+    :param event: EventClass which will be converted into a json object
+    :type event: EventClass
+    :param data_marshaller: Callable function which will cast event.data
+        into a json object
+    :type data_marshaller: typing.Callable
+    :returns: json object representing the given event
+    """
+    return to_structured_http(event, data_marshaller=data_marshaller)[1]
```

Comment:
At some point, I'd expect to invert this logic and have `to_structured_http` call `to_json`. 😁